src/main/java/com/github/pedrovgs/androidwifiadb/view/ConnectDisconnectRenderer.java

```diff
@@ -0,0 +1,27 @@
+package com.github.pedrovgs.androidwifiadb.view;
+
+import java.awt.Component;
+
+import javax.swing.JPanel;
+import javax.swing.JTable;
+import javax.swing.table.TableCellRenderer;
+
+public class ConnectDisconnectRenderer extends JPanel implements TableCellRenderer {
+    private ConnectDisconnectPanel connectDisconnectPane;
```

Comment:
Can't you declare this attribute as final and also initialize it inline? Doing this we can remove the constructor.